presto-docs/src/main/sphinx/connector/thrift.rst

```diff
@@ -0,0 +1,87 @@
+================
+Thrift Connector
+================
+
+Thrift Connector makes it possible to integrate with external storage systems without a custom Presto connector implementation.
```

Comment:
We don't treat connectors as proper nouns, so this should be "The Thrift connector" (use "the" and lowercase "connector")